src/main/java/com/airsquared/blobsaver/Shared.java

```diff
@@ -88,30 +96,35 @@ protected Void call() throws Exception {
                         try {
                             response = makeRequest(new URL("https://api.github.com/repos/airsquared/blobsaver/releases/latest"));
                         } catch (FileNotFoundException ignored) {
-                            return null;
+                            newUnreportableError("Could not check for updates. " +
+                                    "Looks like either blobsaver has been abandoned " +
+                                    "or the GitHub project has been moved elsewhere");
+                            return null; //TODO: find the use of this
```

Comment:
It's just the way JFX services work, there is no use of that

Reply:
All right, I'll remove the comment